test/System.IdentityModel.Tokens.Jwt.Tests/JwtSecurityTokenHandlerTests.cs

```diff
@@ -55,6 +55,31 @@ public void JwtSecurityTokenHandler_CreateToken_SameTypeMultipleValues()
                 Assert.NotNull(aTypeClaims.SingleOrDefault(c => c.Value == value));
         }
 
+        [Fact]
+        public void JwtSecurityTokenHandler_CreateToken_AddShortFormMappingForRsaOAEP()
+        {
+            AppContext.SetSwitch(JwtSecurityTokenHandler._enableRsaOaepMappingSwitch, true); //Set to false to test the default behavior and adjust the expected values accordingly.
```

Comment:
I think another test case with the default behavior would be better than having to adjust an existing one.

Reply:
Done, Thanks Westin!